src/components/TokenInput.tsx

```diff
@@ -20,6 +20,7 @@ interface Props {
   onChange?: (value: string) => void
   disabled?: boolean
   readonly?: boolean
+  showUSDprice?: boolean
```

Comment:
if you're defaulting showUSDprice then do you still need optional flag on this type?

Reply:
Because we are using this component on withdraw and deposit page, so I defined this value as option to avoid modify other files.